packages/babel-helper-create-class-features-plugin/src/fields.ts

```diff
@@ -1103,6 +1110,10 @@ export function buildFieldsInitNodes(
         return injectSuperRef;
       };
 
+  const classRefForInnerBinding =
```

Comment:
When `ref` is not defined and `classRefFlags.ForInnerBinding` is not enabled, e.g. compiling a plain class `class C {}` without private methods / static elements, we will generate a uid identifier but we do not insert it to the final AST. In this case the memoizer variable `_class0`, `_class1`, etc. is not continuous.